include/userobjects/PICStudyBase.h

```diff
@@ -36,19 +42,39 @@ class PICStudyBase : public RayTracingStudy
   ///@}
   /// Ray data for storing the number of real particles each ray represents
   const RayDataIndex _weight_index;
+  /// Ray data for storing the charge of the particle
+  const RayDataIndex _charge_index;
+  /// Ray data for storing the mass of the particle
+  const RayDataIndex _mass_index;
+  /// Ray data for storing the type of physical particle a ray represents
+  const RayDataIndex _species_index;
 
   /// the velocity updater object which we will hold the rules for how our
   /// particles velocities are updated
-  const VelocityUpdaterBase & _velocity_updater;
+  const ParticleStepperBase & _stepper;
 
+  /// temporary variables used when resetting rays
+  ///@{
+  const Elem * _temp_elem;
+  Point _temp_point;
+  Real _temp_distance;
+  Point _temp_velocity;
```

Comment:
`temp` can sometimes be used for temperature, so using the full name could avoid confusion. 
```suggestion
  const Elem * _temporary_elem;
  Point _temporary_point;
  Real _temporary_distance;
  Point _temporary_velocity;
```

Reply:
Left a comment in the source; these should be removed